packages/react-native-win32/src/RNTester/js/utils/RNTesterList.win32.ts

```diff
@@ -54,6 +54,10 @@ const ComponentExamples: Array<IRNTesterExample> = [
     key: 'ImageWin32Test',
     module: require('../../../Libraries/Image/Tests/ImageWin32Test'),
   },
+  {
```

Comment:
You'll need to add anything in src/ to an "override manifest". This can be done by using `npx override add {file}`

Reply:
Hmm I'm getting the following error when running that command:

PS D:\repos\react-native-windows-fork> npx override add D:\repos\react-native-windows-fork\packages\react-native-win32\src\Libraries\Image\Tests\SvgWin32Test.tsxD:\repos\react-native-windows-fork\packages\react-native-win32\src\Libraries\Image\Tests\SvgWin32Test.tsx
npx: installed 1 in 1.322s
Missing Override environment, specify it via the -e command line switch or OVERRIDE_ENV environment variable

Any ideas?